client/src/components/Nav/index.js

```diff
@@ -1,8 +1,8 @@
 import {h, Component} from 'preact';
+import {get_sign_in_out} from "../../js/auth-helpers";
 import style from './style';
 
-import Menu from 'preact-material-components/Menu';
-import Button from 'preact-material-components/Button';
+import { Menu, Button } from 'preact-material-components';
```

Comment:
Yep I know you went this route because of a testing issue, however there is nothing special about the button or menu coming from material components at this point we can get this fairly easily with vanilla css/js especially considering how few instances we are going to have. Sorry for pointing you to preact-material-components to begin with.
in addition just for everyone's info what I have discovered in working on this topic is that importing the component features as an object like this vs the individual imports bloats surprisingly heavy on any component library not just material components. It had the same effect with semantic-ui-react as well which is targeted as a version specific for this environment(react-preact etc) so lesson here beware of the hidden cost of unnecessary libraries

Reply:
I don't mind getting rid of material components - I didn't really spend any time with them on this issue other than editing the import button to get tests to pass - and I noticed how huge the change made the import with that plugin. And I didn't spend much time with them in the first place - just copied and pasted some code I found. I didn't want to change this issue into a styling issue so I just left it and figured we'd refactor later. However, to instantly avoid too huge of a bundle I am taking them out today and adding a lightweight dropdown menu to Nav and lightweight forms for signin and register - none of them requiring any libraries. I'll leave it to other people to style or I'll play around later with smaller libraries to make it look nicer.